src/include/duckdb/main/settings.hpp

```diff
@@ -763,14 +793,33 @@ struct LockConfigurationSetting {
 	static Value GetSetting(const ClientContext &context);
 };
 
-struct LogQueryPathSetting {
+struct LoggingLevel {
 	using RETURN_TYPE = string;
-	static constexpr const char *Name = "log_query_path";
```

Comment:
Can we keep this setting and then turn on the logger to log similarly to the old query path logger?

Reply:
Or for now we can leave the old logger in-tact and leave replacing it for a future PR

Reply:
I've restored it for now, will remove again when I add the file log storage